static/skin/viewer.js

```diff
@@ -310,6 +310,17 @@ function blockLink(url) {
        : url;
 }
 
+function isExternalAppUrl(url) {
+  const EXTERNAL_APP_PROTOCOLS = ['mailto', 'tel', 'geo', 'magnet', 'zoomus'];
```

Comment:
I personally wouldn't hardcode a list of URI schemata given that there are an infinite number of possible schemata, since webapps can define them arbitrarily. If we were only dealing with Wikimedia ZIMs, a hardcoded list might be appropriate, but for Zimit we could have literally any protocol, e.g. `skype:`, `bingmaps:`, `comgooglemaps:`, `comgooglemaps-x-callback:`, etc., etc., etc....? I think we need a different approach: have a list of URI schemata ("protocols") that we know can or must be opened in the app (I only found `javascript:` and `about:`), and open all the rest in a new tab or window.

Reply:
Addressed in a fixup commit